src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+spring.mail.host=smtp.gmail.com
+spring.mail.port=587
+spring.mail.username=$MAIL_ACCOUNT_USERNAME
+spring.mail.password=$MAIL_ACCOUNT_PASSWORD
+
+spring.mail.properties.mail.smtp.auth=true
+spring.mail.properties.mail.smtp.connectiontimeout=5000
+spring.mail.properties.mail.smtp.timeout=5000
+spring.mail.properties.mail.smtp.writetimeout=5000
+spring.mail.properties.mail.smtp.starttls.enable=true
+
+spring.jackson.serialization.write_dates_as_timestamps=false
+
+spring.jpa.hibernate.ddl-auto=update
+spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3306/academy_jdbc_homework
```

Comment:
Not part of JDBC task

Reply:
if I delete this configuration from **application.properties** I cannot run the application
What should I do?

Reply:
It means that you made application in wrong way.
You should not use JdbcTemplate / DataSource in this task.